Header header
uint32 width
uint32 height
float32[] data
